### PR TITLE
fix: renovate update openresty

### PR DESCRIPTION
**Pull request by fzipi**
- openresty can have versions with up to 4 numbers
- it doesn't depend on nginx version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Examples:
 * nginx – *latest stable ModSecurity v3 on Nginx 1.25.3 official stable base image, and latest stable OWASP CRS 4.1.0*
    * [nginx](https://github.com/coreruleset/modsecurity-crs-docker/blob/master/nginx/Dockerfile)
    * [nginx-alpine](https://github.com/coreruleset/modsecurity-crs-docker/blob/master/nginx/Dockerfile-alpine)
-* Openresty - *last stable ModSecurity v3 on Nginx 1.25.3 official stable base image, and latest stable OWASP CRS 4.1.0*
+* Openresty - *last stable ModSecurity v3 on OpenResty 1.25.3 official stable base image, and latest stable OWASP CRS 4.1.0*
```
Comment (theseion) [inline README.md:50]:
```suggestion
* Openresty - *last stable ModSecurity v3 on OpenResty 1.25.3.1 official stable base image, and latest stable OWASP CRS 4.1.0*
```

File: docker-bake.hcl [hunk 2]
```diff
@@ -179,7 +179,6 @@ target "openresty-alpine-fat" {
     dockerfile="openresty/Dockerfile-alpine"
     args = {
         OPENRESTY_VERSION = "${openresty-version}"
-        NGINX_VERSION = "${nginx-version}"
```
Comment (theseion) [inline docker-bake.hcl:182]: Can you derive the nginx version here instead?

Comment (theseion) [reply docker-bake.hcl:182]: And then also add `NGINX_VERSION` as `ARG` to the Dockerfile.

**Review by theseion — CHANGES_REQUESTED** [review]

File: docker-bake.hcl [hunk 3]
```diff
@@ -179,7 +179,7 @@ target "openresty-alpine-fat" {
...
+        NGINX_VERSION = patch(openresty-version)
```
Comment (theseion) [inline docker-bake.hcl:182]: Won't this just be `3` if openresty version is `1.25.3.1`?

Comment (fzipi) [reply docker-bake.hcl:182]: Nope, see https://github.com/coreruleset/modsecurity-crs-docker/blob/5039d9dce5689087d4d0b032476cb671f3e3913e/docker-bake.hcl#L83.

We get all the numbers up to the patch.

**Pull request merged**